src/pages/Landing/index.tsx

```diff
@@ -18,6 +19,7 @@ import { useAppSelector } from 'state/hooks'
 import styled, { css } from 'styled-components/macro'
 import { BREAKPOINTS } from 'theme'
 import { useIsDarkMode } from 'theme/components/ThemeToggle'
+import { TRANSITION_DURATIONS } from 'theme/styles'
 import { Z_INDEX } from 'theme/zIndex'
 
 const PageContainer = styled.div<{ isDarkMode: boolean }>`
```

Comment:
Background removal seems unrelated but if it was meant to be removed, looks like `isDarkMode` is no longer used by this component